Cargo.toml

```diff
@@ -13,12 +13,13 @@ python-source = "pysrc"
 [dependencies]
 bincode = { version = "1.3.3" }
 chrono = { version = "0.4", features = [ "serde" ] }
-chrono-tz = { version = "0.6", features = [ "serde" ] }
+# chrono-tz = { version = "0.6", features = [ "serde" ] }
 futures = { version = "0.3.21" }
 log = { version = "0.4" }
-pyo3 = { version = "0.17.1" }
-pyo3-chrono = { version = "0.5.0" }
-pyo3-log = { version = "0.7.0" }
+# pyo3 = { version = "0.17.1", git = " }
+pyo3 = { path = "../../../repos/pyo3/" }
```

Comment:
I'm currently using a branch on my own fork because the current PR needs a modified version of `chrono` that was not pushed anywhere. I proposed a fix on the PR to make it work with the current version of `chrono`, but the original author prefers to fix things in `chrono` and leave the PR as it is, so I'm waiting to know where this is going first.

I'm keeping this PR in draft until I know more, and then will link to the specific commit once it's usable.

CI would not pass yet anyway, I'm fixing a problem with the Python->Rust conversion right now, trying to instance a `TumblingWindowConfig` with a `start_at` parameter fails.